frontend/lab/samples/forloops/forsequence.evy

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this intentionally an empty file? Not completely sure I understand it's purpose so maybe it's just needed as a placeholder :shrug:

Reply:
yeah, it is needed so I don't get a 404 on frontend side. I should just add a little extra config attribute in sample.json... noted. next time.